src/modules/timepix3/readout/DataParser.h

```diff
@@ -33,7 +33,7 @@ namespace Timepix3 {
 #define PIXEL_DCOL_OFFSET   52
 #define PIXEL_SPIX_OFFSET   45
 #define PIXEL_PIX_OFFSET    44
-#define PIXEL_TOA_OFFSET    28
+#define PIXEL_TOA_OFFSET    30
```

Comment:
I'm wondering  if this should have been caught in a unit test?

Reply:
It should, but the unit test was also incorrect made after the code assume the code is correct. It was a mistake during the project handover.